service/history/mutableStateUtil.go

```diff
@@ -193,6 +193,17 @@ func convertUpdateSignalInfos(
 	return outputs
 }
 
+func convertInt64SetToSlice(
```

Comment:
Should we just pass it down as a set?  It would remove the need for this conversion which does not add much value.

Reply:
actually good idea
I would prefer removing all conversion within mutable state in one PR

Reply:
Can you create an issue and resolve this conversation.

Reply:
#977